### PR TITLE
upgrade selenium to 3.141.59

### PR DESCRIPTION
**Pull request by jtnord**
Upgrades selenium to `3.141.59` 

I don't actually need this - but in the process of actually trying to work out what the issue I was observing was I tried using the latest - turns out the issue was with the docker container & env I was using to run firefox.

so no urgency, but would probably need doing at some point so filing so that the work does not have to be duplicated.

### CODE REVIEW

**Comment by olivergondza** [comment]
Thanks, @jtnord! Can you also update https://github.com/jenkinsci/acceptance-test-harness/blob/master/src/main/java/org/jenkinsci/test/acceptance/FallbackConfig.java#L132-L134 so the containers match the version?

**Comment by jtnord** [comment]
> Can you also update...

all this time I am manually launching a container and setting `BROWSER=remote-webdriver-firefox` and `JENKINS_LOCAL_HOSTNAME` , never even knew that was an option   :-)

File: src/main/java/org/jenkinsci/test/acceptance/utils/ElasticTime.java [hunk 1]
```diff
@@ -59,7 +59,16 @@ public long seconds(long secs) {
     }
 
     public long milliseconds(long ms) {
-        double actualSeconds = concurrency * factor;
+        double actualSeconds = getSlowDownFactor();
         return Math.round(ms * actualSeconds);
     }
+    
+    /**
+     * Get the factor by which we slow down time.
```
Comment (amuniz) [inline src/main/java/org/jenkinsci/test/acceptance/utils/ElasticTime.java:67]: _If my calculations are correct, when this baby hits eighty-eight miles per hour… you’re gonna see some serious shit._

**Doc, “Back To The Future”**

**Comment by olivergondza** [comment]
@jtnord, it is relatively recent addition...

**Comment by olivergondza** [comment]
Thanks, James!

**Pull request merged**